app/components/Field/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { createElement } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
We are ready to use Flow instead of proptypes

Reply:
Done